### DISCUSSION TITLE
Dual engine

### DISCUSSION THREAD

**Discussion by john33138** [hatlabs/discussions#23] [Q&A] [answered]
Hello, I am using a usb sh-esp32 to retrieve all the information from my nmea 2000 network.  Having 2 engines the sh-esp32 defines me all in * .port. *. example: propulsion.port.alternatorVoltage. The starboard does not appear to be present on the network, It is recognized by a LOWRANCE.
Thanks for your help.
![pgn_2](https://user-images.githubusercontent.com/48172957/147883977-528614df-1ffc-4568-a3f2-f49b73791c68.jpg)

**Comment by mairas** [comment, marked as answer]
Check the NMEA 2000 input plugin settings. I believe there's a mapping from the N2K instance numbers to SK path names. It's likely that only the first engine instance (0) has been defined and mapped to `port`. If that's the case, add another instance (probably 1) and map it to e.g. `starboard`.

**Comment by john33138** [comment]
thank you MAIRAS for the information, I will try with my little knowledge to find where the first engine is mapped .
